packages/eslint-plugin-orbit-components/docs/rules/default-theme.md

```diff
@@ -0,0 +1,70 @@
+# default-theme
+
+The aim of this rule is to prevent a wrong usage of the defaultTheme object from @kiwicom/orbit-components.
+
+## Rule details
+
+The following patterns are considered errors:
+
+```jsx
+import defaultTheme from "@kiwicom/orbit-components/lib/defaultTheme";
+
+const failedStatus = {
+  isActive: true,
+  activeColor: defaultTheme.orbit.paletteOrangeNormal,
+  node: <CloseCircle color="warning" />,
+  content: getContentPerStatus(activeStatus, true),
+};
+```
+
+```jsx
+import defaultTheme from "@kiwicom/orbit-components/lib/defaultTheme";
+
+const Wrapper = styled.div`
+  color: ${defaultTheme.orbit.paletteProductNormal};
+`;
+```
+
+```jsx
+import theme from "@kiwicom/orbit-components/lib/defaultTheme";
+```
+
+```jsx
+import defaultTheme from "@kiwicom/orbit-components/lib/defaultTheme";
+
+const {
+  borderWidthCard,
+  borderStyleCard,
+  borderColorCard,
+  borderRadiusNormal,
+  spaceMedium,
+} = defaultTheme.orbit;
+```
```

Comment:
Could you provide a short explanation and an example proper usage after each bad pattern you listed?

For example, I don't understand why destructuring is problematic, although I assume it's because `defaultTheme` should be used only as a whole, and destructuring makes it harder for us to track all these variables and print out warnings for them, so we just mark destructuring itself as a problem?

Reply:
Yes, I can, but in a few words: it's an anti-pattern, in our FE repo I saw that this example is copied from there, users tend to destructure it and pass it later to SC.

Reply:
Ok, I understand.